app/src/main/java/com/example/musicsharing/retrofit/api/BackendApi.kt

```diff
@@ -10,14 +15,47 @@ import retrofit2.http.POST
 import retrofit2.http.Path
 
 interface BackendApi {
+    // User Functions
+    @GET("user/follower")
+    fun followUser(
+        @Body() followPayload: FollowPayload
+    ): Call<ResponseBody>
+
+    @GET("user/spotify/{spotifyId}")
+    fun getUser(
+        @Path("spotifyId") spotifyId: String
+    ): Call<User>
 
     @POST("user/register")
     fun saveUserInfo(
         @Body() userInfo: UserInfoPayload
     ): Call<ResponseBody>
 
-    @GET("user/exists/{spotifyId}")
+    @GET("user/{spotifyId}/exists/")
     fun userExists(@Path("spotifyId") spotifyId: String): Call<Boolean>
 
+    // Post Functions
+
+    @POST("post/comment")
+    fun createComment(
+        @Body() commentPayload: CommentPayload
+    ): Call<ResponseBody>
+
+    @POST("post")
+    fun createPost(
+        @Body() postPayload: PostPayload
+    ): Call<ResponseBody>
```

Comment:
This endpoint doesn't return anything but I can make it so that it returns the created post if you then want to just append that to the list of posts for the feed.